test/import/fixtures/utf8-lazy.esm.default.js

```diff
@@ -0,0 +1 @@
+import { c as ɵɵbar } from './chunk-5275c9cc.js';\u0275\u0275bar();
```

Comment:
I find it a bit surprising that the import itself uses `ɵɵbar` but the function call uses `\u0275\u0275bar`. I understand it works, just find it odd.

Reply:
I also wonder if the renaming back to `ɵɵbar` is necessary at all. The original declaration was renamed to `c` anyway.